docs/deployment/cumulus_api_launchpad_authentication.md

```diff
@@ -5,11 +5,12 @@ hide_title: true
 ---
 
 # Cumulus API Launchpad Authentication
-Cumulus API supports Launchpad as an authentication option.
+Cumulus API supports Launchpad as an authentication option.[^1]
```

Comment:
Super nitpick: there should be a blank line between markdown headings and content